tests/config_validator/test_validate.py

```diff
@@ -0,0 +1,85 @@
+# pylint: disable=missing-function-docstring,missing-module-docstring
+
+import os
+from pathlib import Path
+
+from jsonschema import ValidationError
+import pytest
+
+from tests.__fixtures__.example_schema import EXAMPLE_SCHEMA
+from validator.config.config import Configuration
+from validator.errors.errors import InvalidTargetPathError
+import validator.validate as config_validator
+
+
+def test_build_app_path_with_env(monkeypatch):
```

Comment:
Is it required to have the argument sometimes named `monkeypatch` and sometimes `mocker` ?
I would personally find it clearer if we always used `mocker` but I don't know if this is a common practice.

Reply:
The concepts are similar but the monkeypatching is new to me, here's a link that helps to clarify the difference [here](https://stackoverflow.com/a/47101194/9492453). With that said, I will rename it to `mock` given I'm mocking and not monkeypatching